packages/lisk-transactions/src/base_transaction.ts

```diff
@@ -0,0 +1,263 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+// tslint:disable-next-line no-reference
+/// <reference path="../../../types/browserify-bignum/index.d.ts" />
```

Comment:
Is the reference path here correct? I thought #912 means that each package will reference it's own interface declaration file. Isn't that the case?

Reply:
this is the same path used in the [PR](https://github.com/LiskHQ/lisk-elements/commit/2714828036d50ad7f03fc0563b65f29308285ab9) for the issue.